app/src/main/java/com/sopt/kakaobank/data/api/HomeApiService.kt

```diff
@@ -0,0 +1,17 @@
+package com.sopt.kakaobank.data.api
+
+import com.sopt.kakaobank.core.util.KeyStorage.ACCOUNT_LIST
+import com.sopt.kakaobank.core.util.KeyStorage.API
+import com.sopt.kakaobank.core.util.KeyStorage.V1
+import com.sopt.kakaobank.data.dto.response.ResponseAccountDto
+import retrofit2.http.GET
+import retrofit2.http.Path
+
+interface HomeApiService {
+
+    @GET("$API/$V1/$ACCOUNT_LIST/{memberId}")
+    suspend fun getAccountList(
+        @Path("memberId") memberId: Int
```

Comment:
```suggestion
    @GET("$API/$V1/$ACCOUNT_LIST/{MEMBER_ID}")
    suspend fun getAccountList(
        @Path("MEMBER_ID") memberId: Int
```
이렇게 하고 keyStorage로 뺄 수 있어요!